core/survey/nodeDefExpressionNativeProperties.js

```diff
@@ -0,0 +1,29 @@
+import * as NodeDef from '@core/survey/nodeDef'
+import * as Validation from '@core/validation/validation'
+import SystemError from '@core/systemError'
+
+const nativeProperties = {
+  length: 'length',
+}
+
+const nativePropertiesEvaluators = {
+  length: ({ node }) => {
+    if (Array.isArray(node)) {
+      return node.length
+    }
```

Comment:
length exists also in string objects. we should be able to let users use native js properties. is this coming in another pull request ?

Reply:
yes, it will be done in another pull request